misago/conf/defaults.py

```diff
@@ -39,6 +39,11 @@
 MISAGO_POST_SEARCH_FILTERS = []
 
 
+# Bleach callbacks for linkifying paragraphs
+
+MISAGO_BLEACH_CALLBACKS = []
```

Comment:
Could we get it between `MISAGO_MARKUP_EXTENSIONS` and `MISAGO_POST_VALIDATORS`? Seems more related than to search filters IMHO